frontend/web/product-card/scripts/main.js

```diff
@@ -0,0 +1,243 @@
+
+if (!customElements.get('product-info')) {
+    customElements.define(
+        'product-card',
+        /**
+         * A product card where site visitors will be able to view a featured product and
+         * click on a call to action.
+         * I just implemented the non negotiables:
+         * - Image Source Sets
+         * - Call To Action
+         * Missing Features that will be nice to have when we commit to the job:
+         * - Tap Targets
+         * - Product Variants
+         * 
+         * @property string data-product-title - The product's title displayed as an h2
+         * @property string data-product-content - Text content that displays below the title
+         * @property string data-product-price - Use the money filter before setting this. The product's price displayed below the content.
+         * 
+         * @property string (optional) data-badge-text - Displayed above the title, use this in tandem with data-badge-color-scheme-id.
+         * TODO: Other properties 
+
+         * @property string (optional) data-style-orientation - Accepts image-on-left|image-on-right. Defaults to image-on-left which displays the product image to the left
+         * 
+         * @children sourceset - TODO
+         */
+        class ProductCard extends HTMLElement {
+            isBordered = true; // TODO: Make configurable later
+            orientation = 'image-on-left';
+            title = 'Select a product to use this section';
+            content = '';   // Don't put a placeholder here since what if the product really doesn't have content.
+            price = null;
+
+            // Looked for a royalty free image, for those cases where editors choose an archived product
+            imageUrl = 'https://img.freepik.com/free-photo/sustainability-concept-with-blank-geometric-forms-growing-plant_23-2148994242.jpg';
+            imageAlt = 'Royalty free placeholder image from freepik.com';
+            imageSrcSet = null;
+            imageSizes = null;
+
+            badgeText = null;
+            badgeBgColor = '#5CC1DE';
+            badgeTextColor = '#FFF';
+
+            callToActionText = 'Order Now';
```

Comment:
On second thought, do I really want to put defaults like this? What about locale?